performance/CMakeLists.txt

```diff
@@ -64,7 +66,7 @@ traccc_add_library( traccc_performance performance TYPE SHARED
    "include/traccc/performance/throughput.hpp"
    "src/performance/throughput.cpp" )
 target_link_libraries( traccc_performance
-   PUBLIC traccc::core traccc::io covfie::core detray::test_utils )
+   PUBLIC traccc::core traccc::io covfie::core detray::test_utils dfelibs::dfelibs)
```

Comment:
🤔 The problem is not here. You should not add `dfelibs::dfelibs` here.

The problem is rather that `traccc::io` exposes some CSV specific classes in its public interface. Yet it links against `dfelibs::dfelibs` privately. You should change that to a public dependency. Here:

https://github.com/acts-project/traccc/blob/main/io/CMakeLists.txt#L84-L85

After you change that file, this list of libraries should not be changed anymore.

Reply:
OK